components/article/Article.tsx

```diff
@@ -24,17 +20,15 @@ const StyledArticle = styled.article`
 
 const Innhold = styled.div`
     display: block;
-
-    @media all and (min-width: 601px) {
+    padding: 2rem 1.5rem 4rem 1.5rem;
+    @media all and (min-width: 960px) {
```

Comment:
Har du testa noe på åssen mediaqueries-ene oppfører seg med økt font-size i browser? jeg mener å huske at px kun funker ved +/- zooming, at at em er bedre for media-queries :)

Reply:
Takk for innspill, det skal jeg sjekke. :)

Reply:
Jeg testet litt med økt fontstørrelse vs +/- zooming nå. Virker som om det meste oppfører seg ganske greit med mediaquerise-ene vi har i px. Jeg kan sjekke litt mer over sommeren på om vi bør bytte ut med em/rem i stedet. :)